### PR TITLE
Docker compose folder structure refactoring and README update

### PR DESCRIPTION
**Pull request by cosletmihai**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .PHONY: test install lint generate-tests-from-features
 
-include .env.dev
+include compose/local/api/.dev
```
Comment (costezki) [inline Makefile:3]: this seems an odd place of the env file.

File: Makefile [hunk 2]
```diff
@@ -14,9 +14,9 @@ install-prod:
 	@ pip install -r requirements.txt
 
 install-dev:
-	@ echo "$(BUILD_PRINT)Installing the development requirements"
+	@ echo "$(BUILD_PRINT)Installing the local requirements"
 	@ pip install --upgrade pip
-	@ pip install -r requirements/dev.txt
+	@ pip install -r requirements/local.txt
```
Comment (costezki) [inline Makefile:19]: I wonder whether the local/remote vocabulary is more suggestive than the dev/prod.

File: Makefile [hunk 3]
```diff
@@ -28,44 +28,44 @@ test:
 
 build-dev-api:
 	@ echo -e '$(BUILD_PRINT)Building the api container'
-	@ docker-compose --file docker-compose.dev.yml --env-file .env.dev build validator-api
+	@ docker-compose --file local.yml --env-file compose/local/api/.dev build validator-api
```
Comment (costezki) [inline Makefile:31]: so this already goes into the direction we have been discussing of organising several Docker files in a folder structure.

File: README.md [hunk 4]
```diff
@@ -1,41 +1,94 @@
-# rdf-validator-ws
+# Meaningfy's RDF validator
```
Comment (costezki) [inline README.md:1]: Perhaps we want to omit the company name. But thank you for considering it. Much appreciated.  
We shall however start thinking of a company website indeed.

File: README.md [hunk 5]
```diff
@@ -1,41 +1,94 @@
...
-- schemas 1..*
+## Authors and acknowledgement
+Written by: [Meaningfy](https://github.com/meaningfy-ws)
 
-### Output:
-- Machine readable: Validation Report RDF
-- Human friendly: eds4jinja over the RDF output, resulting in HTML/PDF content
\ No newline at end of file
+Validator tools used:
+- [TopQuadrant](https://github.com/TopQuadrant/shacl)
```
Comment (costezki) [inline README.md:93]: TopQuadrant is probably out of context here. But Mentioning RDFUnit (the tool name) and the AKSW research group (the creator) is with mentioning if not already done so early in the tool description.

File: README.md [hunk 6]
```diff
@@ -1,41 +1,94 @@
-# rdf-validator-ws
+# Meaningfy's RDF validator
 
-Validate your RDF using a web-service API
+An RDF web based validator created by Meaningfy.
+It works by using the work of the guys from [TopQuadrant](https://github.com/TopQuadrant/shacl) with their SHACL validation
```
Comment (costezki) [inline README.md:4]: No SHACL validator is used as provided by TopQuadrant. RDFUnit exists before SHACL was invented and the technology is completely different. SHACL emulation has been implemented from scratch by the ASKW, with some limitation (i.e. a few complex SHACL operations are not handled, but the rest works).

```
RDF validator is a generic web service for validating RDF data with SHACL shape definitions. It is based on the [RDFUnit](https://github.com/AKSW/RDFUnit) developed at the University of Leipzig. 
```

File: usecase_description.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+## Use Cases
```
Comment (costezki) [inline usecase_description.md:1]: well, I guess we no longer need this document. Or perhaps we do :) Not sure.

**Review by costezki — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (cosletmihai) [reply Makefile:19]: to me it seems more natural

**Review by costezki — APPROVED** [review]

**Pull request closed without merging**